### ISSUE TITLE
Suggestion: Allow file URLs for complexmessage file

### ISSUE THREAD

**Issue by CupOfTea696** [jonas747/yagpdb#1006]
I'm trying to have the bot post an image via a custom command, but when I use the code below, it posts a .txt file with the URL rather than the actual image.

```go
{{ sendMessage nil (complexMessage "file" "https://i.imgur.com/xueJWEW.png" }}
```

<img width="344" alt="Screenshot 2021-07-17 at 10 11 12" src="https://user-images.githubusercontent.com/7327050/126032246-a58d4f1b-3acf-4876-a5cb-aaeee951e0ed.png">

**Comment by CupOfTea696** [comment]
Side note: I know I can use an embed to show an image, but this significantly reduces the size of the image in the discord interface

**Comment by CupOfTea696** [comment]
I can probably manage a PR for this, but I'm not sure how you want to handle the difference between text and an actual file URL, given that there may be a case where someone actually wants to create a text file containing just a URL.

**Comment by LemmeCry** [comment]
You could just send the URL with sendMessage and let discord auto-format it into an image

**Comment by CupOfTea696** [comment]
That would show the image link above it tho, not as clean. Oh well.

**Comment by LemmeCry** [comment]
Point being that getting images from the URLs, parsing and uploading them as attachments seems very resource intensive.

**Comment by CupOfTea696** [comment]
Yeah, that's fair enough. I didn't realise the Discord API doesn't allow you to provide a file URL for attachments.
